beacon_node/eth2_libp2p/src/discovery/mod.rs

```diff
@@ -81,6 +82,17 @@ struct SubnetQuery {
     retries: usize,
 }
 
+impl SubnetQuery {
+    /// Returns true if this query has expired.
+    pub fn expired(&self) -> bool {
+        if let Some(ttl) = self.min_ttl {
+            ttl < Instant::now()
+        } else {
+            true
```

Comment:
IIRC the case that we dont have a min_ttl is for long-lived subnets. Are we sure we want these to always be expired. I would have thought they would never expire. I know this is copied from previous code, but is the previous code correct?

Reply:
Ahh nice catch.